app/components/sections/route-plan-blueprints/route-plan-blueprint-table/component.js

```diff
@@ -14,7 +14,7 @@ export default Ember.Component.extend({
     return reg.test(routePlanBlueprint.get("name"));
   },
 
-  @computed("routePlanBlueprints.[]", "filterTerm")
+  @computed("routePlanBlueprints.@each.{name}", "filterTerm")
```

Comment:
I don't think we need this key here since it is never referenced in this.

Also, I think when looping, we should use an abbreviation that is initials of the object.

`i` -> `rpb` I think is clearer when scanning